packages/react-component-library/src/components/SelectE/SelectE.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+import { SelectE, SelectEProps } from './index'
+import { SelectEOption } from './SelectEOption'
+
+export default {
+  component: SelectE,
+  title: 'Select (Experimental)',
+  parameters: {
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      description: {
+        component:
+          'This component wraps a popular open-source library. See comprehensive documentation [here](https://github.com/JedWatson/react-select#readme).',
+      },
+    },
+  },
+} as Meta
+
+const Template: Story<SelectEProps> = (args) => (
+  <SelectE label="Some label" {...args}>
```

Comment:
Would be good to add some height (like for the Select stories) so that it's a bit more usable on the Docs tab:

![image](https://user-images.githubusercontent.com/66470099/143892637-5c4a896e-326f-4fe1-a9c7-1d6446a1d71d.png)

(even better if could escape the iframe, but probably more difficult...)